jplag/src/main/resources/de/jplag/messages.properties

```diff
@@ -6,7 +6,9 @@ CommandLineArgument.ExcludeFile=All files named in this file will be ignored in
 CommandLineArgument.Language=Select the language to parse the submissions
 CommandLineArgument.MinTokenMatch=Tunes the comparison sensitivity by adjusting the minimum token required to be counted as a matching section. A smaller <n> increases the sensitivity but might lead to more false-positives
 CommandLineArgument.ResultFolder=Name of the directory in which the comparison results will be stored
-CommandLineArgument.RootDirectory=Root-directory that contains submissions
+CommandLineArgument.RootDirectory=Root-directory with submissions to check for plagiarism
+CommandLineArgument.PlagiarismDirectory=Root-directory with submissions to check for plagiarism
+CommandLineArgument.PriorDirectory=Root-directory with prior submissions to compare against
```

Comment:
What is the difference between the root directory and the plagiarism directory? Is it just whether --new was explicitly specified or not?

Reply:
Indeed, "root directory" is the previous behavior `jplag dir1 dir2 dir3`. I didn't forbid that to keep backwards compatibility.

Reply:
The `new` option thus exists as opposite of `old` which may be more intuitive to users, and for allowing freedom in the order of directories.

A consequence is that you can now specify no root directories since all 3 ways are optional, but that gets caught by having nothing to check.